CHARTER.md

```diff
@@ -0,0 +1,44 @@
+# ContainerSSH Charter
+
+ContainerSSH is a **free, open source project, and must remain so**. Our main mission is to improve upon ContainerSSH with the goal to create containers for SSH users without having to set up system users.
+
+This charter covers **important processes and decisions that impact the whole organization, direction, and goals**. Small decisions like coding style per repository or similar details can be decided among the repository maintainers.
+
+## Working Group
+
+The working group *(WG)* consists of:
+* 3 chairs with equal rights who
+  - Enforce the [Code of Conduct](CODE_OF_CONDUCT.md)
+  - Have administrative rights over the GitHub org, communication channels, and other community-owned resources
+  - Publicly record decisions made by the WG in the [wg-decisions](wg-decisions) folder
+* Any individual who wants to join and passes a review of their PR sent to [WG_MEMBERS.md](WG_MEMBERS.md))
```

Comment:
I would suggest making the membership criteria more explicit:

To join the working group, you must be one of the following:

- an active user of ContainerSSH (end user, product builder, researcher, etc; please describe your use case in the PR)
- actively helping community members on any of the official ContainerSSH channels
- make code, documentation, or artwork contributions to ContainerSSH